loadtest/control/gencontroller/state.go

```diff
@@ -134,3 +140,41 @@ func copyThreadInfo(src *ThreadInfo) *ThreadInfo {
 	dst.TeamId = src.TeamId
 	return dst
 }
+
+func orderUsers(user1, user2 string) (string, string) {
+	if user1 < user2 {
+		return user1, user2
+	}
+
+	return user2, user1
+}
```

Comment:
This feels more of a util, not state specific, I'd move it to the util file.

Reply:
It's definitely a util, but right now very specific to the userDMs field and only used there. I think I'd keep it here until other code path needs to use it.